### PR TITLE
react events

### PR DESCRIPTION
**Pull request by james-aguirre**
Screenshot 2023-05-19 at 2 04 36 PM
Screenshot 2023-05-19 at 2 05 01 PM

### CODE REVIEW

File: react-events/my-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import logo from './logo.svg';
+import './App.css';
+import CustomButton from './CustomButton';
+function App() {
```
Comment (RobertGardner) [inline react-events/my-app/src/App.js:4]: Please put a blank line after the last import statement.

File: react-events/my-app/src/CustomButton.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import './CustomButton.css';
+
+export default function CustomButton({ text, className, message }) {
+  function handleClick() {
```
Comment (RobertGardner) [inline react-events/my-app/src/CustomButton.js:4]: This is correct for the first phase of the exercise. Now do the steps near the end that have you move this function into App.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
First step fine, next step still to go.

**Comment by james-aguirre** [comment]
Screenshot 2023-05-20 at 7 02 24 PM

File: react-events/my-app/src/CustomButton.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
 import './CustomButton.css';
 
-export default function CustomButton({ text, className, message }) {
-  function handleClick() {
-    window.alert(message);
-  }
+export default function CustomButton({
+  text,
+  className,
+  message,
```
Comment (RobertGardner) [inline react-events/my-app/src/CustomButton.js:6]: You don't use this anymore so you should remove it.

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**